.github/workflows/lint.yaml

```diff
@@ -4,39 +4,17 @@ on: [ push, pull_request ]
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-versions: [ 1.16.x, 1.17.x, 1.18.x, 1.19.x ]
+        platform: [ ubuntu-latest, windows-latest ]
+    runs-on: ${{ matrix.platform }}
     steps:
+      - name: Install Go
```

Comment:
super minor, but your editor indents list items and mine doesn't.  Can you make it the same whitespace as test.yaml, please?